python_frank_energie/frank_energie.py

```
@@ -19,251 +45,999 @@ class FrankEnergie:
+                """,
+                "operationName": "Login",
+                "variables": {"email": username, "password": password},
+            }
+
+            response = await self._query(query)
+            auth_data = None
+            if not response is None:
+                data = response["data"]
+                if not data is None:
+                    auth_data = data["login"]
+                    user_data = auth_data["user"]
```

Ruff

Local variable `user_data` is assigned to but never used

Remove assignment to unused variable `user_data`

(F841)